content/en/docs/reference/faq.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "FAQ"
+linkTitle: "FAQ"
+weight: 5100
+description: >	
+    FAQ.
```

Comment:
The FAQ in the description should be removed (Just `description:`  is ok)and FAQ should be the last item in reference
![image](https://user-images.githubusercontent.com/18525465/174795360-1bb75e44-8e38-4986-a35e-66628c0b57fa.png)